app/models/list_entry.rb

```diff
@@ -0,0 +1,4 @@
+class ListEntry < ActiveRecord::Base
+  belongs_to :list
+  belongs_to :book
```

Comment:
It's likely we will needs that, but I would prefer to hold off until we get to a story which requires that association.